src/libs/fileDownload/index.js

```diff
@@ -0,0 +1,39 @@
+import openURLInNewTab from '../openURLInNewTab';
+import getAttachmentName from './getAttachmentName';
+
+/**
+ * Downloading attachment in web, desktop
+ * @param {String} url
+ */
+export default function fileDownload(url) {
+    fetch(url)
+        .then(response => response.blob())
+        .then((blob) => {
+            // Create blob link to download
+            const href = URL.createObjectURL(
+                new Blob([blob]),
+            );
+
+            // creating anchor tag to initiate download
+            const link = document.createElement('a');
+
+            // adding href to anchor
+            link.href = href;
+            link.setAttribute(
+                'download',
+                getAttachmentName(url), // generating the file name
+            );
+
+            // Append to html link element page
+            document.body.appendChild(link);
+
+            // Start download
+            link.click();
+
+            // Clean up and remove the link
+            link.parentNode.removeChild(link);
```

Comment:
also from the davidwalshblog looks like he has a call to `URL.revokeObjectURL` https://developer.mozilla.org/en-US/docs/Web/API/URL/revokeObjectURL

Reply:
Okay, will add this. seems good to add this.

Reply:
Added the change